functests/test_suite_test.go

```diff
@@ -24,10 +32,24 @@ func TestTest(t *testing.T) {
 	if junitPath != nil {
 		rr = append(rr, reporters.NewJUnitReporter(*junitPath))
 	}
-	RunSpecsWithDefaultAndCustomReporters(t, "Test Suite", rr)
+	RunSpecsWithDefaultAndCustomReporters(t, "Performance Addon Operators e2e tests", rr)
 }
 
 var _ = BeforeSuite(func() {
-	//clients.Setup()
-	// Add here the setup for additional features
+	cs := testclient.New("")
```

Comment:
Sorry I just noticed, how about making this global and using it in all the tests?
In that case I would have a longer (and more descriptive) name since the scope is wider.

Reply:
what name do you prefer?

Reply:
clientSet maybe?

Reply:
hm maybe instead of global variable we can provide getter for the clientset
```
var clientSet *ClientSet

func Get() *ClientSet {
    if clientSet == nil {
        clientSet = New()
    }
    return clientSet
}
```
WDT?

Reply:
We risk races here I think? Don't remember if ginkgo tests are executed in parallel.
On the other hand, I wouldn't waste a mutex here.

Another option could be calling New() in the package's init() func and having the ClientSet variable exposed. 

something along the line of 
```
package client

var ClientSet ClientSet

func init() {
    // init here
}
```

On the calling site it would just be `client.ClientSet`.

Reply:
done